docs/guides/publish.md

```diff
@@ -13,6 +13,23 @@ build it by default. This means that your project may not be ready for distribut
 the effect of declaring a build system in the
 [project concept](../concepts/projects.md#build-systems) documentation.
 
+!!! note
+
+    If you have internal packages that you do not want to be published, you can mark them as
+    private:
+
+    ```toml
+    [project]
+    classifiers = ["Private :: Do Not Upload"]
+    ```
+
+    uv will still attempt to publish packages with the classifier, but PyPI will reject them. Note
+    that the private classifier is used against PyPI, but when using an alternative registry,
+    access to the uploaded packages needs to still be configured correctly.
```

Comment:
I'm sorry to continue picking at this, but I'm earnestly confused by the language here. "is used against PyPI" doesn't quite make sense, the classifier is sent to all indexes and only PyPI is expected to act on it right? What does it mean to configure access to the uploaded packages on an alternative registry? I thought the goal here was to avoid upload and publish entirely — I'm confused that we're talking about access controls now.